.storybook/stories/TableWithRemoteData.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { Fragment, useRef, useState } from 'react';
```

Comment:
I've moved this table with remote data component out of `Pagintaion` component and into the new `Examples` section.

Reply:
Lovely stuff